src/code42cli/errors.py

```diff
@@ -124,6 +105,14 @@ def invoke(self, ctx):
         except click.exceptions.Exit:
             raise
 
+        except (
+            UserDoesNotExistError,
+            Py42UserAlreadyAddedError,
+            Py42InvalidRuleOperationError,
+        ) as err:
+            self.logger.log_error(err)
+            raise LoggedCLIError(str(err))
```

Comment:
I think these should be re-raised as plain `Code42CLIError`'s since there isn't any additional data that's going to be logged that isn't already in the error text itself.

Reply:
You're right, nothing in the log that provides anymore info